recipe/meta.yaml

```diff
@@ -12,29 +12,29 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 0
+  number: 1
   script: {{ PYTHON }} -m pip install . --no-deps -vv
   noarch: python
 
 requirements:
   host:
-    - python >=3.5
+    - python >=3.8
     - pip
-    - setuptools >=38.6.0
+    - setuptools >=64.0.0
     - setuptools_scm >1.5.4
   run:
-    - python >=3.5
+    - python >=3.8
     - asciitree
-    - numpy >=1.7
+    - numpy >=1.20
     - fasteners
     - numcodecs >=0.10.0
 
 test:
   requires:
     - pytest
-    # Needs `pkg_resources` in the tests
-    # xref: https://github.com/zarr-developers/zarr-python/pull/813
-    - setuptools
+    # Needs `packaging` in the tests
+    # xref: https://github.com/zarr-developers/zarr-python/pull/1450
+    - packaging
```

Comment:
Noticed this changed in PR ( https://github.com/zarr-developers/zarr-python/pull/1450 ). So refreshed the dependency (and comment)

Reply:
Ah it's not in 2.15 🤦‍♂️

Will revert and add to the release PR

Reply:
Adding here ( https://github.com/conda-forge/zarr-feedstock/pull/80#discussion_r1270089929 )